cluster/get-kube.sh

```diff
@@ -183,6 +193,9 @@ if [[ ${KUBE_VERSION} =~ ${KUBE_CI_VERSION_REGEX} ]]; then
   # Override KUBERNETES_RELEASE_URL to point to the CI bucket;
   # this will be used by get-kube-binaries.sh.
   KUBERNETES_RELEASE_URL="${KUBERNETES_CI_RELEASE_URL}"
+elif [[ ${KUBE_VERSION} =~ ${KUBE_BAZEL_VERSION_REGEX} ]]; then
+  KUBE_VERSION=${KUBE_VERSION#bazel/}
```

Comment:
Sounds fine to me. I only jumped through all of these hoops because the existing pattern was very tight regex validation of the release string. If we add an option to disable the regex matching altogether, then that should be the only change that I need.

What do you think, @ixdy? You have a lot of commits to this script specifically to keep these regexes up to date and accurate.